docs/telescopes/scopus.md

```diff
@@ -10,81 +10,49 @@ proceedings." -- [SCOPUS website](https://www.elsevier.com/solutions/scopus).
 The telescope will look for connections conforming to the Airflow connection ID naming convention (below) and generate a
 subdag to handle the entire ETL pipeline for each institution.
 
-## Airflow Connection ID naming
+## Observatory Platform API
 
-Each connection id will have a name of the form ```scopus_<institution>```, for example ```scopus_curin```. 
+The telescope relies on the Observatory Platform API in order to create dags. A DAG will be created in Airflow for every scopus telescope returned by the API, i.e., one for each organisation.
 
-### Attributes / extra fields
-The extra field of the Airflow connection should be a json parsable dictionary with the following keys. 
-```
-"api_keys": [list of dictionaries of the form {"key": "keystring", (optionally) "view": "standard or complete"]
-"start_date":  Python pendulum or datetime parseable date string
-"id": [list of scopus institution id strings],
-"project_id": "optional override for project_id",
-"transform_bucket_name": "optional override for transform_bucket_name",
-"download_bucket_name": "optional override for download_bucket_name",
-"data_location": "optional override for data_location"
-```
-For example:
-```
-{
-  "api_keys" : [ {"key": "test_key", "view": "standard"}, {"key": "another_key"}],
-  "start_date" : "2020-09-01",
-  "id" : "60031226",
-  "project_id": "new_project_id",
-  "transform_bucket_name": "new_transform_bucket_name",
-  "download_bucket_name": "new_download_bucket_name"
-}
-```
-
-## DAG definition
-The main dag will check that the API server is up.  If this is the case, each subdag will proceed to execute its ETL
-pipeline.
-
-## Subdag flow
-```
-check_dependencies >> download >> upload_downloaded >> transform_db_format >> upload_transformed >> bq_load >> cleanup
-```
+The following fields need to be set in the extra field of the telescope:
+ * `airflow_connections` which is a list of Airflow connection ID names with API keys set in the password field.
+ * `institution_ids` which is a list of strings containing the institution IDs to search in SCOPUS, e.g., ['60031226'] for Curtin University.
+ * `earliest_date` which is the earliest `datetime` to query.
 
-### check_dependencies
 
-This checks the airflow configuration variables (including overrides), and configures some release parameters.
+## Storage location
 
-### download
+The telescope saves the dataset to a Google BigQuery table with the `project_id` specified in the standard Airflow variable for the project ID. The `dataset_id` defaults to `elsevier`. The `table_id` is set to `scopus<date suffix>`.
```

Comment:
If you were fetching data for multiple institutions with a telescope each, would they all get written into the same table?